docs/pages/includes/database-access/database-config.yaml

```diff
@@ -23,6 +23,8 @@ db_service:
 
   # Matchers for registering Azure-hosted databases.
   azure:
+    # Set is_flexi_server to true when using an Azure flexible server.
+    is_flexi_server: false
```

Comment:
I didn't notice you had put this under the azure matchers section in the original PR.

This setting is only for static database config `databases:` below. So we'll need to update master as well

Example:

```
  databases:
    - name: "teleport-mysql-flex-static"
      azure:
        is_flexi_server: true
      uri: "teleport-mysql-flex.mysql.database.azure.com:3306"
      protocol: "mysql"
      static_labels:
        env: "dev"
```

Reply:
Hotfix for master: #21237 - commit has been cherry-picked here.